app/assets/javascripts/editor.js

```diff
@@ -23,36 +31,36 @@ $E = {
         return code;
       }
     });
-  },
-  setState: function(textarea = 'text-input', preview = 'comment-preview-main', title = 'title') {
+  }
+  setState = function(textarea = 'text-input', preview = 'comment-preview-main', title = 'title') {
     $E.title = $('#' + title + 'title'); // not sure why this exists? seems like $E.title is always #title
     $E.textarea = $('#' + textarea);
     $E.textarea.bind('input propertychange', $E.save);
     $E.preview = $('#' + preview);
-  },
-  is_editing: function() {
-    return ($E.textarea[0].selectionStart == 0 && $E.textarea[0].selectionEnd == 0)
-  },
-  refresh: function() {
+  };
+  // code seems unused, commenting out for now.
+  // is_editing = function() {
+  //   return ($E.textarea[0].selectionStart == 0 && $E.textarea[0].selectionEnd == 0)
+  // };
```

Comment:
Couldn't find this code used anywhere doing a search for `is_editing` in VSCode. Will delete in the next few PRs if the tests keep passing.

Reply:
Let's remove and clean this then 💯

Reply:
Deleted it in another PR!